testing/test_many.out

```diff
@@ -2,12 +2,12 @@ Running testing/test_many.sh
 Many Tests
 DAQ stress test
 Enough results: 1
-Enough DHCP timeouts: 1
+Enough DHCP timeouts: 0
 Enough static ips: 1
 Found NTP and DNS traffic for static ip devices: 1 1
-Enough ipaddr tests: 1
-Enough alternate subnet ips: 1
-Enough ipaddr timeouts: 1
+Enough ipaddr tests: 0
```

Comment:
this doesn't seem right... why are all these changing?  Are all of these expected changes or did they just happen?